jax/_src/random.py

```diff
@@ -957,7 +958,7 @@ def _cond_fn(kXVU):
     true_fun = lambda: lax.ge(lax.log(U), lax.add(lax.mul(X, one_over_two),
                                                       lax.mul(d, lax.add(lax.sub(one, V),
                                                                          lax.log(V)))))
-    false_fun = lambda: False
+    false_fun = lambda: lax.le(one, zero)
```

Comment:
What's the reason for turning a constant into an expression? Mabye `lambda: np.bool_(False)` would be sufficient?

Reply:
No good reason. Fixed :)